pkg/arvo/sys/vane/clay.hoon

```diff
@@ -4245,6 +4245,21 @@
           %-  wrap:fusion
           (page-to-cage:(aeon-ford yon) u.peg)
         ``cage+[-:!>(*^cage) cage]
+      ::
+          %subs
+        =/  doj=(unit dojo)  (~(get by dos.rom) i.t.pax)
+        ?~  doj  ~
```

Comment:
I think this might be better as `[~ ~]`.  We know the answer -- there are no subscribers.  `~` is for when we don't know the answer.